app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java

```diff
@@ -158,6 +158,8 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
                 .appendPattern("yyyy-MM-dd")
                 .toFormatter(Locale.US);
 
+        parseSwipeData(cafe.swipeData());
```

Comment:
hmm, does this not return anything? or where does the data go?

Reply:
Updated this for better styling. The method parseSwipeData basically updated an instance variable but changed it to return a List of Swipes